pages/ksphere/kommander/latest/operations/access-control/index.md

```diff
@@ -6,36 +6,30 @@ menuWeight: 1
 excerpt: Centrally manage access across clusters
 ---
 
-Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters. The resources are similar to Kubernetes RBAC but with some crucial differences.
+Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters or a subset of clusters. The resources may seem familiar because they are similar to Kubernetes RBAC but with some crucial differences that make it possible to define the roles and policies once and have them federated to clusters within a given scope:
 
-## Types of Access Control Objects
-
-**Groups** map to groups and user claims from your identity providers.
-
-![Groups](/ksphere/kommander/img/Access-control-groups-table.png)
+- Global or Workspace-level roles and policies federate those resources to cluster-scoped roles and policies. In the case of workspace-level resources, they only apply to workspace clusters.
```

Comment:
Once we have Workspace-level capitalized and once not capitalized. Shouldn't it be consistent?

Reply:
I agree, i think it should probably be workspace-level. (But who ever is the expert, please let me know!)